server/views/layout.njk

```diff
@@ -121,8 +121,21 @@
 
 {% block bodyEnd %}
   {# Run JavaScript at end of the <body>, to avoid blocking the initial render. #}
-  <script type="module" src="/assets/govukFrontendInit.js"></script>
+  <script type="module" src="/assets/ext/day.js"></script>
+  <script type="module" src="/assets/ext/dayjs/plugin/customParseFormat.js"></script>
+  <script type="module" src="/assets/ext/chart.js"></script>
+  <script type="module" src="/assets/ext/chartjs-datalabels.js"></script>  
```

Comment:
Presumably we don't need to load all this on every page across the app? 
Is this just required on pages that show dashboards?

If this is in their documentation can you get them to update it?

Reply:
The guidance specifies to add the scripts to the nunjucks layout but I'll double check this with them